waterlog/public/index.html

```diff
@@ -37,5 +43,6 @@
       To begin the development, run `npm start` or `yarn start`.
       To create a production bundle, use `npm run build` or `yarn build`.
     -->
-  </body>
-</html>
+</body>
+
+</html>
```

Comment:
empty line EOF

Reply:
Done.
With this one we didn't even touch it.